### DISCUSSION TITLE
Selecting audio track in editor freezes game

### DISCUSSION THREAD

**Discussion by jkrumpholc** [ppy/osu#23921] [Q&A]
### Type

Crash to desktop

### Bug description

Steps to reproduce:
Open osu! lazer
Go to edit
Click on audio track
Game freezes
Needs to be killed by Task Manager

Specs: W10 21H1, AMD Ryzen 7 5800H, Nvidia 3060 Laptop

### Screenshots or videos

https://github.com/ppy/osu/assets/40388573/b2ee93ee-9387-4088-9ce7-a369bf95af7a

### Version

2023.6141.lazer

### Logs

[database.log](https://github.com/ppy/osu/files/11755563/database.log)
[input.log](https://github.com/ppy/osu/files/11755565/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11755566/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/11755567/network.log)
[performance.log](https://github.com/ppy/osu/files/11755568/performance.log)
[updater.log](https://github.com/ppy/osu/files/11755570/updater.log)
[runtime.log](https://github.com/ppy/osu/files/11755569/runtime.log)

**Comment by peppy** [comment]
what disks do you have connected to your pc? this is going to be something related to your disk setup.

**Reply by jkrumpholc** [reply]
Sorry for late reply

I have 2 disks. First one (disk C) is M.2 type about 2 y/o and I have Windows & Osu Lazer here (type: Kingston OM8PDP3512B-AA1)

Second one (disk D) is 2.5" SSD, about 5 y/o (I have there "old" Osu with my beatmaps skins etc. which were imported to Lazer) (type: Kingston SA400S37480G)

None of the Lazer data should be on D
